src/AccessibilityInsights/App.config

```diff
@@ -9,6 +9,7 @@
       <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.1" />
   </startup>
   <runtime>
+    <AppContextSwitchOverrides value="Switch.UseLegacyAccessibilityFeatures=false;Switch.UseLegacyAccessibilityFeatures.2=false;Switch.UseLegacyAccessibilityFeatures.3=false;Switch.UseLegacyToolTipDisplay=false" />
```

Comment:
You can find this in "Tooltips on keyboard access" at this link: https://docs.microsoft.com/en-us/dotnet/framework/whats-new/whats-new-in-accessibility#wpf48